### ISSUE TITLE
Metrics comparison between YOLO26l and YOLO11l

### ISSUE THREAD

**Issue by edosez-siteassist** [ultralytics/ultralytics#23581]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hello everyone!

I've been testing YOLO26l against YOLO11l and I got improved mAP performance, which is extremely appreciated. Nonetheless, when looking at validation metrics such as box loss and class loss, those are considerably higher for YOLO26 (have a look at attached screenshots). 

master = yolo11
workspace = yolo26

## Validation losses

<img width="436" height="364" alt="Image" src="https://github.com/user-attachments/assets/bda68a09-e668-4042-8e5f-3296e025fff4" />

<img width="436" height="364" alt="Image" src="https://github.com/user-attachments/assets/2736f5f0-159e-4b37-98b9-dd0301ecb89a" />

<img width="436" height="364" alt="Image" src="https://github.com/user-attachments/assets/11a7e0fd-f0d6-42f7-b792-3af997efac32" />

_Note: DFL has been removed from YOLO26 so I think the chart is no longer relevant, is this correct?_

## Performance charts

<img width="443" height="364" alt="Image" src="https://github.com/user-attachments/assets/837c3225-4fe5-4751-993f-f5c09f659b17" />

<img width="443" height="364" alt="Image" src="https://github.com/user-attachments/assets/27a828e4-08ac-47b1-940e-57ac8d47648e" />

<img width="443" height="364" alt="Image" src="https://github.com/user-attachments/assets/7f5c009e-0dc5-4bf8-a628-de087d8e7ea3" />

The model performance looks better but the loss metric point towards less precise bounding boxes (box loss). I'm also slightly confused at class loss being higher but mAP improving. 

I'm aware of YOLO26's architectural change compared to previous versions: suppression of DFL, introduction of ProgLoss in place of CIoU + DFL so the comparison may not really hold, I'm looking for confirmation if that's the case.

Got these conclusion from Gemini

## Impact on Evaluation Metrics (Loss vs. mAP)

- **BCE + CIoU + DFL**: Typically produces lower raw loss values on charts because DFL explicitly optimizes the discretized boundary distributions, making the mathematical "error" look smaller.

- **ProgLoss**: Focuses on maximizing the final mAP (mean Average Precision) rather than just minimizing raw mathematical loss. This can lead to higher raw validation losses (box and class loss) even while achieving a higher mAP because the model is optimized for ranking and detection accuracy rather than boundary-probability precision.

Can anyone help me clarify whether this comparison makes sense?

Thank you

**Comment by UltralyticsAssistant** [comment]
👋 Hello @edosez-siteassist, thank you for your interest in Ultralytics 🚀! We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

Thanks for sharing the detailed context + screenshots—this is a great comparison topic between YOLO26l and YOLO11l 🤝📊. Loss curves can be tricky to interpret across architectures/criteria, so having a bit more run context will help the team give a precise explanation.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

To help us compare apples-to-apples, please also include:
-Exact `yolo` command or Python snippet used for each run (YOLO11l vs YOLO26l)
-`ultralytics` version(s), commit/branch (you mention `master = yolo11`, `workspace = yolo26`)
-The dataset YAML and number of classes/images
-Key hyperparameters: `imgsz`, `batch`, `epochs`, `lr0`, augmentations, `rect`, `multi_scale`, etc.
-Whether the validation is being run with identical settings for both models
-Your full `results.csv` (or the relevant rows) and the training/val console logs around validation

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

This is an automated response 🤖—an Ultralytics engineer will also assist soon.

**Comment by Y-T-G** [comment]
Losses are not metrics that are comparable across different architectures. You can get higher or lower loss just by increasing or decreasing the weights of the loss or changing the loss function.

They are not comparable with YOLO11 because YOLO26 loss is a weighted sum of losses from two heads.

**Comment by edosez-siteassist** [comment]
> Losses are not metrics that are comparable across different architectures. You can get higher or lower loss just by increasing or decreasing the weights of the loss or changing the loss function.
> 
> They are not comparable with YOLO11 because YOLO26 loss is a weighted sum of losses from two heads.

Thank you for clarifying it @Y-T-G
